worlds/sc2/options.py

```diff
@@ -270,13 +270,12 @@ class EnableRaceSwapVariants(Choice):
     designed for. NOTE: Cutscenes are always skipped on race-swapped mission variants.
 
     Disabled: Don't shuffle any non-vanilla map variants into the pool.
+    Pick One: Shuffle up to 1 valid version of each map into the pool, depending on other settings.
     Shuffle All: Each version of a map can appear in the same pool (so a map can appear up to 3 times as different races)
-    ("Pick Just One At Random" coming soon)
     """
     display_name = "Enable Race-Swapped Mission Variants"
     option_disabled = 0
-    # TODO: Implement pick-one logic
-    # option_pick_one = 1
+    option_pick_one = 1
```

Comment:
`option_pick_one` sounds like it'll just pick one race-swapped mission to add to your pool by glance. For example, I would think this means it gives me a singular race-swapped mission instead of preventing duplicates (if i skipped over the tooltip). I suggest to rename it to `option_shuffle_no_duplicates` to follow the pattern of `option_shuffle_all`. Shuffle no duplicates would imply it is shuffled in, but have no duplicate maps.

Reply:
I think "pick one" is clear enough in-context, and the description on the tooltip is sufficient to answer questions.